jekyll/_cci2/building-docker-images.md

```diff
@@ -59,24 +62,25 @@ jobs:
       # build and push Docker image
       - run: |
           TAG=0.1.$CIRCLE_BUILD_NUM
-          docker build -t   CircleCI-Public/circleci-demo-docker:$TAG .      # (3)
-          docker login -u $DOCKER_USER -p $DOCKER_PASS         # (4)
+          docker build -t   CircleCI-Public/circleci-demo-docker:$TAG .      # (4)
+          docker login -u $DOCKER_USER -p $DOCKER_PASS         # (5)
           docker push CircleCI-Public/circleci-demo-docker:$TAG
 ```
 
 Let’s break down what’s happening during this build’s execution:
 
 1. All commands are executed in the [primary container][primary-container].
-2. Once `setup_remote_docker` is called, a new remote environment is created, and your primary container is configured to use it.
-3. All docker-related commands are also executed in your primary container, but building/pushing images and running containers happens in the remote Docker Engine.
-4. We use project environment variables to store credentials for Docker Hub.
+2. Once `setup_docker_engine` is called, a new remote environment is created, and your primary container is configured to use it.
+3. https://circleci.com/docs/2.0/docker-layer-caching/
```

Comment:
Would it be rendered just as is or replaced with page title? I mean we can add more user-friendly text, i.e. "Read more about Docker Layer Caching".

Reply:
fixed